include/wx/sizer.h

```diff
@@ -223,7 +228,7 @@ class WXDLLIMPEXP_CORE wxSizerFlags
 
 private:
 #ifdef wxNEEDS_BORDER_IN_PX
-    static int DoGetDefaultBorderInPx();
+    static double DoGetDefaultBorderInPx();
```

Comment:
Just curious, you use `double` here and in `wxPrivate::DpiDependentValue<double>` while `float` seems to be used elsewhere for scaling with DPI?

Reply:
`GetContentScaleFactor()` returns double so I kept using that. I can use float if this is preferred.

Reply:
I see, sorry for the noise.

Reply:
It did make me look at the code again, and I noticed `C6011: Dereferencing NULL pointer 'win'.` IntelliSense warning. Every remark can be useful.